doc/dev/doc/toc.md

```diff
@@ -16,9 +19,10 @@
 ## [AtomicLong](atomiclong.md)
 ## [AtomicRef](atomicref.md)
 # [CP SubSystem](cpsubsystem.md)
-# [Locking](locking.md)
-# [Events](events.md)
-# [Kerberos](kerberos.md)
+# [Concepts](concepts.md)
```

Comment:
What is the idea for grouping these three under "Concepts" section?

Reply:
i am trying to structure the doc somehow... these felt like "leftovers" but yea I don't like "concepts" much and then why is serialization not a concept? ok, killing "concepts" for now ;)